src/main/java/project/cinema/service/MovieSessionService.java

```diff
@@ -12,4 +13,6 @@
     void update(MovieSession movieSession);
 
     void delete(Long id);
+
+    Optional<MovieSession> getById(Long id);
```

Comment:
```suggestion
    MovieSession getById(Long id);
```
same - Optional from dao layer, entity from service layer